README.md

```diff
@@ -65,18 +63,18 @@ Soon to be published
 #### Gradle
 
 ```groovy
-compile 'org.glpi:api:1.0.0'
+compile 'org.glpi:api:0.1.0@aar'
 ```
 
 #### Apache Ivy
 
 ```
-<dependency org='org.glpi' name='api' rev='0.1.0'>
+<dependency org='org.glpi' name='api' rev='1.0.0'>
   <artifact name='api' ext='pom' ></artifact>
 </dependency>
 ```
 
-You can also find us on [**Bintray repository**](https://bintray.com/flyve-mdm/). -->
+You can also find us on [**Bintray repository**](https://bintray.com/glpi-project/maven-repository/android-api-client).
```

Comment:
You needs to deploy as `jar` and not `aar` @Ivans51 
Because we can use this lib for Android and any Java project.

Reply:
@Naylin15 you can use the same slug of the repo for the slug on bintray `java-library-glpi`

Reply:
Changed 👍